app/templates/account.hbs

```diff
@@ -3,19 +3,17 @@
 <header>
   {{#unless config.enterprise}}
```

Comment:
I'm not sure, but we can try removing this and testing it out, but we would need to talk to @bnferguson about this too. Might be best to leave it for now.